templates/store/snap-distro-install.html

```diff
@@ -65,16 +65,17 @@
           </g>
         </g>
       </svg>
-      <div class="row">
-        <div class="mobile-col-2 col-8"></div>
-        <div class="mobile-col-2 col-4">
-          <img class="distro-banner__logo" src="{{ distro_logo }}" />
-        </div>
-      </div>
     </div>
     <div class="p-strip">
       <div class="row">
-        <h1 class="p-heading--two u-no-margin is-light">How to install {{ snap_title }}<br/> on <b>{{ distro_name }}</b></h1>
+        <div class="col-8">
+           <h1 class="p-heading--two u-no-margin is-light">How to install {{ snap_title }}<br/> on <b>{{ distro_name }}</b></h1>
+        </div>
+        {% if distro_logo_mono %}
+        <div class="col-4 mobile-col-2">
```

Comment:
Great! Thanks for moving it here.

2 small things we can inprove:
- `mobie-col-2` is not needed (we hide it on mobile)
- to hide it on mobile we can use `u-hide--small` class and avoid custom styles